src/components/ProviderCard.js

```diff
@@ -0,0 +1,284 @@
+import {
+  Fab,
+  Divider,
+  Container,
+  Typography,
+  Box,
+  Card,
+  Stack,
+} from '@mui/material';
+import Grid from '@mui/material/Grid';
+import ReactReadMoreReadLess from 'react-read-more-read-less';
+import { ShortInfo } from './ShortInfo';
+
+import { ArrowForwardIos } from '@mui/icons-material';
+
+import { CheckProfilePic } from '../Utils';
+import { Link } from 'react-router-dom';
+
+export const ProviderCard = ({ provider }) => {
+  const {
+    name,
+    languages,
+    bio,
+    education,
+    title,
+    location,
+    avatarUrl,
+  } = provider || {};
+
+  return (
+    <Container>
+      <Grid container spacing={2} columns={{ xs: 8 }}>
+        <Grid item xs={8} sx={{ mt: 3 }}>
+          <Stack direction="row" alignItems="center" gap={1}>
+            <Typography variant="subtitle1">
+              <Link style={{ textDecoration: 'none' }} to="/">
+                Providers
+              </Link>
+            </Typography>
+            <ArrowForwardIos style={{ fontSize: '12' }} />
+            <Typography variant="subtitle1">{name}</Typography>
+          </Stack>
+        </Grid>
+        <Grid item xs={3} sx={{ mt: 1 }}>
+          <Card sx={{ display: 'flex', height: '350px' }}>
+            <img
+              src={CheckProfilePic(
+                avatarUrl,
+                'https://icon-library.com/images/no-user-image-icon/no-user-image-icon-0.jpg'
+              )}
+              alt={name}
```

Comment:
Nice catch remembering to add alt tags for accessibility! I would improve the alt tag to add "profile image" for screenreader users to know exactly what the image is.